x-pack/plugins/threat_intelligence/README.md

```diff
@@ -16,18 +16,24 @@ Verify your node version [here](https://github.com/elastic/kibana/blob/main/.nod
 
 `yarn es snapshot --license trial`
 
-**Run Kibana:**
+**Enable the plugin:**
+
+Add the following to your `kibana.yml`:
+
+```
+# =================== Security Solution ===================
+# Enable the new Threat Intelligence plugin
+xpack.securitySolution.enableExperimental: ['threatIntelligenceEnabled']
+```
```

Comment:
I had done this in a previous PR but @maxcold suggested instead to link the development setup mdx file that we have in the security-team repo (to have a single source of truth: https://github.com/elastic/security-team/blob/main/docs/protections-team/threat-intelligence-services/protections-experience/development-setup.mdx

I just realized that I put the text in our README but forgot the actual link.
On the main branch, line 23 it says

> See here to get your `kibana.yaml` to enable the Threat Intelligence plugin.

I think we should just add the link there

> See [here](https://github.com/elastic/security-team/blob/main/docs/protections-team/threat-intelligence-services/protections-experience/development-setup.mdx) to get your `kibana.yaml` to enable the Threat Intelligence plugin.

Reply:
personally I'm a fan of having everything in the same place instead of multiple repos, but the fact is that we do have some md files in that security-team repo so why not linking them...

Reply:
yeah, the main thing I want to avoid is having two places where we need to keep the information up to date. Where this place is is the second question to me. I'm fine having the development set up in the repo together with the source code, but then pls move all that's available in the security team docs https://github.com/elastic/security-team/blob/main/docs/protections-team/threat-intelligence-services/protections-experience/development-setup.mdx here, remove it from there and only link to the file in our source code

Reply:
Got you, partners, let me fix that.

Reply:
I have moved the contents of `development-setup` to our internal readme, once this is merged - we will add a reference in there. Here is the issue: https://github.com/elastic/security-team/issues/4959.